pages/ksphere/kommander/latest/operations/cloud-providers/index.md

```diff
@@ -156,6 +156,69 @@ Figure 7 - Cloud Provider Form with values
 
 Once created, a Cloud Provider’s display name or credentials can be updated.
 
+### Configuring an Azure cloud provider
+
+When creating an Azure cloud provider, you need an [ansible inventory file](https://docs.ansible.com/ansible/latest/user_guide/intro_inventory.html) and ssh keys.
+
+#### Create a new set of credentils via CLI commands
+
+```
+az login
+```
+
```

Comment:
I would replace by the description [in my PR](https://github.com/mesosphere/dcos-docs-site/pull/2713/files) as it adds a few details and has been successfully tested by @tillt today:

Find the `SUBSCRIPTION_ID` of your account, named `id` in the output of this command:
```
az account show
```

Create the service principal for the provider:
```
az ad sp create-for-rbac --role="Contributor" --scopes="/subscriptions/SUBSCRIPTION_ID"
```

The command will return data needed to create the secret:
```
{
  "appId": "APP_ID",
  "displayName": "name",
  "name": "http://name",
  "password": "PASSWORD",
  "tenant": "TENANT"
}
```

Reply:
@armandgrillet feel free contributing to the PR :)

Reply:
Armand, are you going to add your description?

Reply:
I added Armand's description